Felienne/hedy
# [BUG FIX] Admin is able to customize own class again

**Description**
Same issue as #3296, same fix, other page.

**Fixes**
No related issue.

**How to test**
Make sure you are an admin and the owner of a class. When attempting to customize this class verify that the buttons are shown as expected.
